en/user_guide/websites_creation.rst

```diff
@@ -0,0 +1,93 @@
+Website creation
+================
+
+Open Orchestra is multisite and allows to create several websites from the same Back Office application.
+
+Name
+----
+
+The field name contributes the name of the website and he allows to generate the identifier of the website.
+
+L'identifiant du site
+---------------------
+
+The site id of field is generated when the name of the site is filled, but it remains alterable during the creation of the site.
+This attribute is unique, once the site is created it is not possible any more to modify it.
+Open Orchestra verifies during the form validation that the site id does not already exist.
+
+Aliases
+-------
+
+Open Orchestra allows to create  several alias for a site.
+
+For example :
+
+* Scheme by default : http
+* Domain : demo.openorchestra.dev
+* Language : English
+* Language prefix : en
+* main aliases : on
+
+The website home page will be ``http://demo.openorchestra.dev/en``.
+
+Open Orchestra use only the main aliases for the sitemap generation.
+
+Blocks available
+----------------
+
+This attribute list all the block available on Open Orchestra.
+The selected blocks will be the ones visible during the node contribution.
+
+Theme by default
+----------------
+
+The attribute theme is the theme by default used for the website.
+It is possible to select an other theme during the node creation.
+
+sitemap generation
+------------------
+
+The two folling attribute are used for the sitemap generation :
+
+* Indicative periodicity of change
+* Relative importance compared to the other pages of the website, this attribute is used by default for the node creation.
+
+Meta
+----
+
+The meta attributes provide information about the nature and content of a web page, they are added in the page HTML header with the use of meta tags.
+These attributes are also provided in the website configuration, which value is used if they are not overriden by the page configuration.
+
+Meta keyword
+~~~~~~~~~~~~
+
+This is a list of keywords asociated to the page and can be used by search engines for ranking.
+
+Meta description
+~~~~~~~~~~~~~~~~
+
+This is used by search engines to display a description of the page in the results list.
+
+Meta index
+~~~~~~~~~~
+
+This attribute defines whether or not the page should be referenced in search engines.
+
+Meta follow
+~~~~~~~~~~~
+
+This attribute indicates to search engines if they should follow the hypertext links on the page in order to index other documents.
+
+Informations contained in the robots.txt file
+---------------------------------------------
+
+This attribute is what will contain the robots.txt of the website.
+
+Example de contribution du robots.txt
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+    User-agent: *
+    Allow: /
+
+User-agent: * mean that one or several instruction (s) which follow applies for all the agents.
+Allow: mean that the search engine can browse all the directories and the pages of the site.
```

Comment:
Maybe explain separately what is Allow (list of paths that are crawlable), and "/" (all pages crawlable)